public/index.html

```diff
@@ -11,6 +11,17 @@
     -->
   <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+
+  <script>
+    !function () {
+      var analytics = window.analytics = window.analytics || []; if (!analytics.initialize) if (analytics.invoked) window.console && console.error && console.error("Segment snippet included twice."); else {
+        analytics.invoked = !0; analytics.methods = ["trackSubmit", "trackClick", "trackLink", "trackForm", "pageview", "identify", "reset", "group", "track", "ready", "alias", "debug", "page", "once", "off", "on"]; analytics.factory = function (t) { return function () { var e = Array.prototype.slice.call(arguments); e.unshift(t); analytics.push(e); return analytics } }; for (var t = 0; t < analytics.methods.length; t++) { var e = analytics.methods[t]; analytics[e] = analytics.factory(e) } analytics.load = function (t, e) { var n = document.createElement("script"); n.type = "text/javascript"; n.async = !0; n.src = "https://cdn.segment.com/analytics.js/v1/" + t + "/analytics.min.js"; var a = document.getElementsByTagName("script")[0]; a.parentNode.insertBefore(n, a); analytics._loadOptions = e }; analytics.SNIPPET_VERSION = "4.1.0";
+        analytics.load("mNZ52QK4RGH7GyKot2UGGyj5lCrS1eNL");
```

Comment:
i'm guessing this is the part of the code that needs to be updated with a new ID once we set up a Segment account for the team.

Reply:
Wondering if we need to do any environment checking, we probably don't want to be logging user data from staging.

Reply:
That is a good consideration. I'll do some testing. I think setting the url in the Segment should filter out the staging visits, but it is possible that the app ID in the JS is enough generate data.

Reply:
After some testing, the analytics are tracked even in development. I added a condition to set the app ID to empty string if host is `localhost` and that works. I tried not loading analytics at all, but then the calls to analytics.js in the React components break the app.